js_modules/dagit/packages/core/src/runs/RunActionsMenu.tsx

```diff
@@ -257,6 +264,9 @@ export const RunBulkActionsMenu: React.FC<{
   const selectedIDs = selected.map((run) => run.runId);
   const deletionMap = selected.reduce((accum, run) => ({...accum, [run.id]: run.canTerminate}), {});
 
+  const failedRuns = selected.filter((r) => failedStatuses.has(r?.status));
+  const reexecutionMap = failedRuns.reduce((accum, run) => ({...accum, [run.id]: run}), {});
+
```

Comment:
For now, just support re-execution from failure. `ALL_OPS` will be added later. cc @johannkm